tests/sdks/python/application/test_envilder_facade.py

```
@@ -0,0 +1,157 @@
+        # Assert
+        call_args = factory_mock.call_args
+        options = call_args[1]["options"]
+        assert options.profile == "staging"
+
+
+class TestEnvilderValidation:
+    def Should_RaiseFileNotFoundError_When_FileDoesNotExist(
+        self,
+    ) -> None:
+        # Act
+        action = lambda: Envilder.load("non-existent.json")
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `action` as a `def`

(E731)